src/bindings.d.ts

```diff
@@ -1,17 +1,11 @@
 export {}
 
 declare global {
-    const _AUTH0_DOMAIN: string
-
-    const _AUTH0_CLIENT_ID: string
-
-    const _AUTH0_REDIRECT_URI: string
-
-    const _AUTH0_URI: string
-
-    const _TWITTER_FETCH_USER_URI: string
-
-    const _TWITTER_BEARER_TOKEN: string
-
+    const AUTH0_DOMAIN: string
```

Comment:
@ankurdotb keeping it like this doesn't actually populate values. It only tells the typescript compiler that these values mean somthing